M2/Macaulay2/packages/Macaulay2Doc/repl.m2

```diff
@@ -969,57 +969,51 @@ document { Key => symbol OutputDictionary,
      ///,
      SeeAlso => { "dictionaryPath" }
      }
-document { Key => Pseudocode,
+document { Key => {Pseudocode, PseudocodeClosure},
      Headline => "the class of pseudocodes",
      "The Macaulay2 interpreter compiles its language into pseudocode, which is evaluated later, step by step.  At each
      step, the evaluator is considering a pseudocode item.  These pseudocode items are normally not available to the user, but
-     the internal function ", TO "disassemble", " can display their contents, the function ", TO "pseudocode", " can convert
-     a function closure to pseudocode, the function ", TO "value", " can evaluate it (bindings of values to local symbols
+     the internal function ", TO "pseudocode", " can convert
+     a function closure to pseudocode and display their contents, the function ", TO "value", " can evaluate it (bindings of values to local symbols
      are enclosed with the pseudocode), the operator ", TO "===", " can be used for equality testing,
      and when the debugger is activated after an error, the variable ", TO "current", " contains the pseudocode step whose execution produced the error.",
      }
 document { Key => pseudocode,
      Headline => "produce the pseudocode for a function",
      Usage => "pseudocode f",
      Inputs => { "f" => FunctionClosure },
-     Outputs => { Pseudocode => { "the pseudocode of the function ", TT "f"} },
-     SeeAlso => { disassemble },
+     Outputs => { PseudocodeClosure => { "the pseudocode of the function ", TT "f"} },
      EXAMPLE lines ///
 	  pseudocode resolution
-	  disassemble oo
-     ///
-     }
-document { Key => disassemble,
-     Headline => "disassemble pseudocode or a function",
-     Usage => "disassemble c",
-     Inputs => { "c" => Pseudocode },
-     Outputs => { String => {"the disassembled form of ", TT "c"} },
-     SeeAlso => { pseudocode },
+     ///,
+     PARA {
+	  "One can look at specific part of the code by using ", TT "_",":"
+	  },
      EXAMPLE lines ///
-     disassemble res
+          oo_4_1_1_0
+	  value oo
      ///,
      PARA {
-	  "It may be useful to disassemble code during debugging, as in the following demonstration."
+	  "It may be useful to look at code during debugging, as in the following demonstration."
 	  },
      EXAMPLE lines ///
      load "Macaulay2Doc/demo1.m2"
      code g
      g 2
      code current
-     disassemble current
+     current
```

Comment:
Same here! (Is this a duplicate?)

Reply:
yeah the code is a duplicate, but that was already the case before.